qiskit_ibm_runtime/runtime_job.py

```diff
@@ -655,3 +656,24 @@ def tags(self) -> List:
             Tags assigned to the job that can be used for filtering.
         """
         return self._tags
+
+    @property
+    def cost_estimation(self) -> Dict[str, Any]:
+        """Return cost estimation infromation for this job.
+
+        Returns:
+            Cost estimation information in dictionary format, with the estimated
+            running time and estimated max running time in seconds.
```

Comment:
What is estimated max running time?

Reply:
I will update the docstrings & release note to be more clear - my understanding is:

`estimated_running_time_seconds`: the value returned by estimation service, estimating how long the job will run
`estimated_max_running_time_seconds`: the estimated running time multiplied by some multiplier based on failure ratio that estimates the absolute maximum the job could run (before it should be canceled?)

Reply:
> before it should be canceled?

Yeah that was my question. Is that the new max execution time? Presumably only used if user didn't specify one?

Reply:
we only need to worry about `estimated_running_time_seconds` as mentioned in my other comment